src/python/pants/binaries/binary_tool.py

```diff
@@ -19,11 +19,15 @@ class BinaryToolBase(Subsystem):
   :API: public
   """
   # Subclasses must set these to appropriate values for the tool they define.
-  # They must also set options_scope to the tool name as understood by BinaryUtil.
+  # They must also set options_scope appropriately.
   support_dir = None
   platform_dependent = None
   default_version = None
 
+  # Subclasses may set this to the tool name as understood by BinaryUtil.
+  # If unset, it defaults to the value of options_scope.
+  name = None
```

Comment:
Is there a test that exercises the behavior described here?

Reply:
In practice yes - buildozer.py requires it, and its tests pass...